README.md

```diff
@@ -1,2 +1,7 @@
 # fall-2020-r1
 Main repository for the first Unity project of Fall 2020
+Testing changes
+
+I added another line, is that better?
```

Comment:
GOOD LINE!!

Reply:
THANK YOU, THANK YOU! I worked so hard on it.

Reply:
This really isn't quite as bad as I thought. With the review rule, we don't have to worry about people accidentally merging into master

Reply:
But can you still push to master? Directly?

Reply:
Nope!